addons/web/static/src/views/graph/graph_renderer.js

```diff
@@ -384,9 +384,9 @@ export class GraphRenderer extends Component {
                 dataset.backgroundColor = hexToRGBA(dataset.borderColor, LINE_FILL_TRANSPARENCY);
             }
             if (cumulated) {
-                let accumulator = 0;
+                let accumulator = dataset.cumulated_start;
                 dataset.data = dataset.data.map((value) => {
-                    accumulator += value;
+                    accumulator += value || 0;
```

Comment:
? the issue is value being wrong (bcs fill_temporal) not the cumulated_start

Reply:
If it worked before it should still work if you have correctly initialize accumulator but maybe you have spotted a bug?

Reply:
I had a bug while writing the original code but I can't find how to reproduce it now. Should I remove it or keep it as it can act as a defensive safeguard?

Reply:
I think you should remove it if you have no good reason to let it. If we find a bug later, we will fix it.